packages/protocol/contracts/L1/libs/LibProvingAlt.sol

```diff
@@ -22,15 +22,16 @@ import "../verifiers/IVerifier.sol";
 import "../TaikoData.sol";
 import "./LibUtils.sol";
 
-/// @title LibProvingAlt
-/// @notice An alternative library for handling block contestation and proving in the Taiko
+/// @title LibProving
```

Comment:
Keeping the file name as LIbProvingAlt is a good idea. Later we can have a simple PR by using `git mv LibProvingAlt.sol LibProving.sol`